thinc/backends/ops.py

```diff
@@ -1442,6 +1525,9 @@ def backprop_lstm_gates(
 
 def sigmoid(X, out=None):
     xp = get_array_module(X)
+
+    # To prevent overflows and help with regularization/numerical stability
+    X = xp.clip(X, -20.0, 20.0)
```

Comment:
It looks like some commits from `master` have crept into the history here. Let me try updating `develop` with the latest from `master`, then you can merge the latest from `develop` back in here and hopefully these should disappear from the diff 🤞

Reply:
`master` have been synced to `develop` so try merging `develop` into this PR once more to see whether that removes these edits?